aiida/cmdline/commands/cmd_rehash.py

```diff
@@ -29,14 +29,35 @@
     default=None,
     help='Only include nodes that are class or sub class of the class identified by this entry point.'
 )
+@options.FORCE()
 @decorators.with_dbenv()
-def rehash(nodes, entry_point):
-    """Recompute the hash for nodes in the database
+def rehash(nodes, entry_point, force):
+    """Recompute the hash for nodes in the database.
 
     The set of nodes that will be rehashed can be filtered by their identifier and/or based on their class.
     """
     from aiida.orm import Data, ProcessNode, QueryBuilder
 
+    if not force:
+        echo.echo_warning('This command will recompute and overwrite the hashes of all nodes.')
+        echo.echo_warning('Note that this can take a lot of time for big databases.')
+        echo.echo_warning('Also, do it only if you are sure of what you are doing.')
```

Comment:
Is the third warning necessary? Apart from it taking long, it cannot do much harm can it? Often we even tell them to do it, such as after a migration.

Reply:
True. I'm still not 100% sure that the hash will be always properly calculated for Floats, but for now I'll remove this warning.